packages/proxy/schema/model_list.json

```diff
@@ -71,6 +49,40 @@
     "max_input_tokens": 400000,
     "max_output_tokens": 128000
   },
+  "gpt-5-pro": {
```

Comment:
demoting from the first pick since we are doing fake responses and our streaming support is lack luster